extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -569,10 +580,45 @@ describes.realWin(
 
           expect(stateValue).to.equal('accepted');
           expect(stateManagerInfo).to.deep.equal({
-            'consentState': 1,
+            'consentState': CONSENT_ITEM_STATE.ACCEPTED,
             'consentString': 'newstring',
             'isDirty': undefined,
-            'consentMetadata': {},
+            'consentMetadata': constructMetadata(
+              CONSENT_STRING_TYPE.US_PRIVACY_STRING
+            ),
+          });
+        });
+
+        it('syncs data with no metadata', async () => {
```

Comment:
Related to discussion in https://github.com/ampproject/amphtml/pull/28028/files#r432607061

IMO the better way to do this is to keep existing tests unchanged, and add one more unit test "syncs data with metadata".

Reply:
Ok